django/db/backends/base/base.py

```diff
@@ -36,6 +37,8 @@ class BaseDatabaseWrapper(object):
     ops = None
     vendor = 'unknown'
     SchemaEditorClass = None
+    # Classes instantiated inside __init__().
+    validation_class = BaseDatabaseValidation
```

Comment:
Sorry about that, I was looking at the imports and forgot they are subclassed in that same file.